.vscode/settings.json

```diff
@@ -93,4 +93,5 @@
 		"**/*.log": true,
 		"**/DS_Store": true,
 	},
+	"search.followSymlinks": false
```

Comment:
Is this intentional?

Reply:
I just find without this line, my mac CPU usage is very high when I open this repo. Not sure if anyone else encounters this issue. Then I find something in web: https://stackoverflow.com/questions/51886037/visual-studio-code-using-large-amounts-of-cpu